torch/csrc/autograd/python_variable.cpp

```diff
@@ -1644,8 +1673,12 @@ PyTypeObject THPVariableType = {
     Py_TPFLAGS_DEFAULT | Py_TPFLAGS_BASETYPE |
         Py_TPFLAGS_HAVE_GC, /* tp_flags */
     nullptr, /* tp_doc */
-    // Also set by metaclass
+// Also set by metaclass
+#if PY_VERSION_HEX >= 0x030B0000
```

Comment:
We can set this unconditionally right?

Reply:
Yes, we can have it for all versions **unconditionally**